### PR TITLE
Add Generic Sensor-based motions sensors

### LINKED ISSUE #629: Add Generic Sensor-based motions sensors
The Generic Sensor-based motions sensors define a modern low-level API that is expected to supersede [Device Orientation](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/deviceorientation/) and [Device Motion](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/devicemotion/) (both supported in Edge):
- Accelerometer / [Established standard (CR)](https://www.w3.org/TR/accelerometer/) / [CanIUse](https://caniuse.com/#feat=accelerometer)
- Gyroscope / [Established standard (CR)](https://www.w3.org/TR/gyroscope/) / [CanIUse](https://caniuse.com/#feat=gyroscope)
- Magnetometer / [Established standard (CR)](https://www.w3.org/TR/magnetometer/) / [CanIUse](https://caniuse.com/#feat=magnetometer)
- Orientation Sensor / [Established standard (CR)](https://www.w3.org/TR/orientation-sensor/) / [CanIUse](https://caniuse.com/#feat=orientation-sensor)

Notes:

* The APIs ship in Chrome 67 and later, See [Chrome Platform Status](https://www.chromestatus.com/feature/5698781827825664)
* Specs at Candidate Recommendation i.e. W3C invites implementations
* As of Aug 2018, global adoption >50% per CanIUse.com
* Some Libraries that depend on the APIs:
  * https://github.com/immersive-web/webvr-polyfill
  * https://github.com/immersive-web/webxr-polyfill
  * https://github.com/kenchris/sensor-polyfills (polyfill)
* Demos:
  * https://intel.github.io/generic-sensor-demos/
* Articles:
  * https://mobiforge.com/design-development/the-generic-sensor-api
  * https://developers.google.com/web/updates/2017/09/sensors-for-the-web

### PR DESCRIPTION
**Pull request by anssiko**
Close #629

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -5248,5 +5248,55 @@
...
+      "text": "Under Consideration"
+    },
+    "spec": "gyroscope",
+    "id": 5698781827825664
+  },
+  {
+    "name": "Magnetometer",
+    "category": "Device",
+    "link": "https://w3c.github.io/magnetometer/",
+    "summary": "Provides access to magnetic field measurements along the device's
```
Comment (alrra) [inline status.json]: Needs an end `",`.

File: status.json [hunk 2]
```diff
@@ -5248,5 +5248,55 @@
...
+      "text": "Under Consideration"
+    },
+    "spec": "magnetometer",
+    "id": 5698781827825664
+  },
+  {
+    "name": "Orientation Sensor",
+    "category": "Device",
+    "link": "https://w3c.github.io/orientation-sensor/",
+    "summary": "Provides access to device’s physical orientation in WebGL-compatible
```
Comment (alrra) [inline status.json]: Needs an end `",`.

File: status.json (hunk 1 above)
Comment (anssiko) [reply status.json]: Fixed (removed the line break).

File: status.json (hunk 2 above)
Comment (anssiko) [reply status.json]: Ditto.

**Comment by alrra** [comment]
@anssiko Can you sign the [CLA](https://github.com/MicrosoftEdge/Status/pull/630#issuecomment-416663776)? Thanks!

**Comment by anssiko** [comment]
@alrra, done!

**Comment by alrra** [comment]
Thank you, @anssiko!

**Pull request closed without merging**